### PR TITLE
Minor UI issues with buttons menus cards localization and responsive design

### LINKED ISSUE #37: Change color: set retention period for datasets
**Describe the bug**
image

**Is this a regression?**
Yes/No.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment**

- Browser [e.g. firefox, chrome]
- Damap version [e.g. 2.1.0]

**Additional context**

### LINKED ISSUE #194: Minor UI Issues with Buttons, Menus, Cards, Localization, and Responsive Design
**Describe the bug**

  Update Buttons Margin - Positioning Issue
        _Problem_: The margin around the update buttons is incorrect, causing half of the button to be hidden when selected.

  Menu Field Positioning Outside of Container
        _Problem_: When the menu field is expanded, it appears outside of its designated container.

  Card Repository Content Position
        _Problem_: Content within card repositories is misaligned.

  Button Footer Styling on Smaller Windows
        _Problem_: Footer buttons do not style correctly on smaller window sizes.

**Expected behavior**

**Screenshots**

**Environment**

- Browser [e.g. firefox, chrome] 
- Damap version [e.g. 2.1.0]

### PR DESCRIPTION
**Pull request by lpandath**
#### What does this PR do?
_Update Buttons Margin - Positioning Issue:_ The margin around the update buttons is fixed to prevent part of the button from being hidden when selected.

_Menu Field Positioning Outside of Container:_ Adjustments are made so that when the menu field is expanded, it remains within its designated container.

_Card Repository Content Position:_ The content within card repositories is realigned for proper display.

_Button Footer Styling on Smaller Windows:_ Changes are implemented to ensure footer buttons style correctly on smaller window sizes.

#### Breaking changes

No

### Checks

- [ ] Summary updated
- [ ] Version view updated
- [ ] Documentation added
- [ ] Tests added
- [ ] E2e tests created
- [ ] Successfully ran e2e tests before merge

closes GH-

### CODE REVIEW

File: libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
     [control]="description"
     label="dmp.steps.project.input.description"></app-textarea-wrapper>
   <button
+    style="margin-left: 5px"
```
Comment (rekt-hard) [inline libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.html]: can we set this on the button element throughout the application? So we do not have to add this everywhere

**Review by rekt-hard — COMMENTED** [review]
Just a small annotation

File: libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.html (hunk 1 above)
Comment (lpandath) [reply libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.html]: yes, let me just check once more if that was needed for every button.

**Review by rekt-hard — APPROVED** [review]
looks good, thank you :)

**Pull request merged**